lib/rubocop/cop/lint/return_in_void_context.rb

```diff
@@ -35,22 +35,15 @@ class ReturnInVoidContext < Base
         def on_return(return_node)
           return unless return_node.descendants.any?
 
-          context_node = non_void_context(return_node)
-
-          return unless context_node&.def_type?
-          return unless context_node&.void_context?
+          method_node = return_node.each_ancestor(:def).first
```

Comment:
Can you rename from `method_node` to `def_node`?

```suggestion
          def_node = return_node.each_ancestor(:def).first
```